spec/vector_spec.rb

```diff
@@ -1966,6 +1966,24 @@ def expect_correct_tokens hash
     end
   end
 
+  context "#sort_by_index" do
+    let(:asc) { vector.sort_by_index }
+    let(:desc) { vector.sort_by_index(ascending: false) }
+
+    context 'numeric vector' do
+      let(:vector) { Daru::Vector.new [11, 13, 12], index: [23, 21, 22] }
+      specify { expect(asc.to_a).to eq [13, 12, 11] }
```

Comment:
Well, I believe it was my suggestion. Personally I prefer to use `it` and `specify` semantically (e.g. what reads better). For this case, "it expect asc to eq" reads strange, while "specify: expect asc to eq" is better. And using `its` is inconvenient, because we need to test two different subjects (asc and desc) in one context, it is not reasonable to split them into even more contexts.